support_for_advertising_use_cases.md

```diff
@@ -66,21 +66,21 @@ The ads industry has endeavored to find good ways of answering these basic quest
 | [Cross Browser / Cross Device Measurement](#cross-browser--cross-device-measurement) | No support | No support | Facebook proposal for “[Cross Browser Anonymous Conversion Reporting](https://github.com/w3c/web-advertising/blob/master/cross-browser-anonymous-conversion-reporting.md)” |
-| [Conversion Fraud](#conversion-fraud) | A section of the [Multi-Browser Aggregation Service Explainer](https://github.com/WICG/conversion-measurement-api/blob/master/SERVICE.md#authenticating-inputs) describes using a service that includes an authentication mechanism. This use case is also being explored for the event level API in this [GitHub Issue](https://github.com/WICG/conversion-measurement-api/issues/13). | Confirmation this is a use-case Safari cares about addressing. In the process of collaboratively designing a solution on this [GitHub Issue](https://github.com/WICG/ad-click-attribution/issues/27). | Facebook proposal for “[Private Fraud Prevention](https://github.com/siyengar/private-fraud-prevention)” |
-| [Click Flooding](#click-flooding) | No solutions yet for this problem. However, lift measurement could be a potential way of measuring this problem. This might be supported by the “[Aggregate Reporting API](https://github.com/csharrison/aggregate-reporting-api)”. | No solutions yet for this problem. | Facebook proposal for “[Private Lift Measurement](https://github.com/w3c/web-advertising/blob/master/private-lift-measurement-conceptual-overview.md)” a potential way of measuring this problem |
+| [Conversion Fraud](#conversion-fraud) | A section of the [Multi-Browser Aggregation Service Explainer](https://github.com/WICG/conversion-measurement-api/blob/master/SERVICE.md#authenticating-inputs) describes using a service that includes an authentication mechanism. This use case is also being explored for the event level API in this [GitHub Issue](https://github.com/WICG/conversion-measurement-api/issues/13). | Confirmation this is a use-case Safari cares about addressing. In the process of collaboratively designing a solution on this [GitHub Issue](https://github.com/WICG/ad-click-attribution/issues/27). | Facebook proposal for “[Private Fraud Prevention](https://github.com/siyengar/private-fraud-prevention)” <br><br> SPARROW : Thanks to exact reporting, it will be possible to link conversion to both an interest group and a publisher. It will also be possible to have user-level granularity on the website, helping to spot fraudulent activity. |
```

Comment:
> SPARROW : Thanks to exact reporting, it will be possible to link conversion to both an interest group and a publisher.

I don't see how this is compatible with the privacy goals of TURTLEDOVE, which the SPARROW explainer says are its goals also.  Your [Privacy Considerations section](https://github.com/BasileLeparmentier/SPARROW#privacy-considerations) says "The Gatekeeper receives interests group x publisher data, but cannot link this data to individual users since it has no user-level information."  Linking to a specific conversion seems to contradict that.

> It will also be possible to have user-level granularity on the website, helping to spot fraudulent activity.

Again this contradicts "The Gatekeeper [...] has no user-level information."  And I'm not sure what "the website" refers to here, but if it's the publisher site then I don't see how to reconcile this with "Web sites cannot learn the interest groups of the people who visit them" — one of the two original privacy goals.

Reply:
Everything here is advertiser side. 
The advertiser is the one getting conversion information, not the gatekeeper. The advertiser can use the click_id in the granular reporting to join to first-party user-level data gathered on his domain.
I will edit the section to make it clearer.

Reply:
OK sure.  I've opened https://github.com/WICG/sparrow/issues/16 to talk about my privacy worries here.